opencti-platform/opencti-front/src/components/ViewSwitchingButtons.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { FunctionComponent } from 'react';
+import ToggleButton from '@mui/material/ToggleButton';
+import Tooltip from '@mui/material/Tooltip';
+import { LibraryBooksOutlined, ViewModuleOutlined } from '@mui/icons-material';
+import { FormatListGroup, Group, RelationManyToMany, VectorPolygon } from 'mdi-material-ui';
+import { ToggleButtonGroup } from '@mui/material';
+import { ListViewIcon, SublistViewIcon } from 'filigran-icon';
+import { makeStyles } from '@mui/styles';
+import FiligranIcon from '@components/common/FiligranIcon';
+import { useFormatter } from './i18n';
+
+const useStyles = makeStyles({
+  EntityIcon: {
+    height: 16,
+    width: 16,
+  },
+  subEntityButton: {
+    height: 36,
+  },
+});
+
+interface ViewSwitchingButtonsProps {
+  handleChangeView?: (value: string) => void;
+  disableCards?: boolean,
+  enableEntitiesView?: boolean,
+  currentView?: string,
+  enableSubEntityLines?: boolean,
+  enableGraph?: boolean,
+  enableNestedView?: boolean,
+  enableContextualView?: boolean,
+}
+
+const ViewSwitchingButtons: FunctionComponent<ViewSwitchingButtonsProps> = ({
+  handleChangeView,
+  disableCards,
+  enableEntitiesView,
+  currentView,
+  enableSubEntityLines,
+  enableGraph,
+  enableNestedView,
+  enableContextualView,
+}) => {
+  const { t_i18n } = useFormatter();
+  const classes = useStyles();
+  return (
+    <>
+      <ToggleButtonGroup
+        size="small"
+        color="secondary"
+        exclusive={true}
+        value={currentView}
+        style={{ margin: '0 0 0 5px' }}
+      >
+        {typeof handleChangeView === 'function' && !disableCards && (
+        <ToggleButton value="cards" aria-label="cards">
+          <Tooltip title={t_i18n('Cards view')}>
```

Comment:
Prefer put the `Tooltip` above the `ToggleButton` otherwise only the icon will have the tooltip, the padding inside the button won't

Reply:
Same for other buttons